engine/tutorials/usingdocker.md

```diff
@@ -121,43 +121,43 @@ Now you can see your running container using the `docker ps` command.
     CONTAINER ID  IMAGE                   COMMAND       CREATED        STATUS        PORTS                    NAMES
     bc533791f3f5  training/webapp:latest  python app.py 5 seconds ago  Up 2 seconds  0.0.0.0:49155->5000/tcp  nostalgic_morse
 
-You can see you've specified a new flag, `-l`, for the `docker ps`
-command. This tells the `docker ps` command to return the details of the
+You can see a new flag, `-l`, for the `docker ps` command.
```

Comment:
Maybe rewrite to something like "The `-l` flag causes `docker ps` to return the details of the _last_ container that was started.